src/MICore/Debugger.cs

```diff
@@ -1126,7 +1126,11 @@ internal void OnComplete(Results results, MICommandFactory commandFactory)
                     {
                         miError = results.FindString("msg");
                     }
-
+                    else
+                    {
+                        miError = results.ResultClass.ToString();
```

Comment:
>results.ResultClass.ToString(); [](start = 34, length = 31)

Shouldn't this be some sort of new localized string like "Unexpected ResultClass from MI Debugger. Expected '{0}' but received '{1}'."